src/mdbook.rs

```diff
@@ -91,7 +91,8 @@ fn handle(
         let (content, content_title) = self::content_with_extract_title(&content);
         // Fallback to summary title, If it is not found in md document
         let title = content_title.unwrap_or_else(|| title(summary, &file.path(), doc_id));
-        file.raw_content_with_content(&title, &content)
+        let content = file.raw_content_with_content(&title, &content);
+        replace_backtick(&content)
```

Comment:
Checkout `avoid_single_use_variables` in https://www.fifthtry.com/fifthtry/handbook/rust-code-guidelines/.